### PR TITLE
chore(scripts): fix "jest:watch" command

### PR DESCRIPTION
**Pull request by layershifter**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

This PR updates existing Gulp tasks to fix them.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/12313)

### CODE REVIEW

File: packages/fluentui/react/gulpfile.ts [hunk 1]
```diff
@@ -5,4 +5,4 @@ import '../../../gulpfile';
 
 // Redefine test and test:watch to build info json files beforehand
 task('test', series('build:docs:component-info', 'test:jest'));
-task('test:watch', series('build:docs:component-info', 'test:jest', parallel('test:jest:watch', 'watch:docs:component-info')));
```
Comment (layershifter) [inline packages/fluentui/react/gulpfile.ts]: We don't need to run `test:jest` before `test:jest:watch`

File: scripts/gulp/tasks/test-unit.ts [hunk 2]
```diff
@@ -53,5 +54,5 @@ if (config.isRoot) {
   task('test:watch', series('test:jest:setup', parallel('test:jest:watch', 'watch:docs:component-info')));
 } else {
   task('test', series('test:jest'));
-  task('test:watch', series('test:jest', 'test:jest:watch'));
+  task('test:watch', series('test:jest:watch'));
```
Comment (layershifter) [inline scripts/gulp/tasks/test-unit.ts]: Same thing here

File: scripts/gulp/tasks/test-unit.ts [hunk 3]
```diff
@@ -9,11 +9,13 @@ const argv = yargs
   .option('runInBand', {})
   .option('maxWorkers', {})
   .option('detectLeaks', {})
+  .option('coverage', { default: true })
```
Comment (layershifter) [inline scripts/gulp/tasks/test-unit.ts]: It was really annoying to run tests without coverage metrics

Comment (silviuavram) [reply scripts/gulp/tasks/test-unit.ts]: I personally don't care to see by default coverage when I run tests, so I would rather have `yarn test:cover` separately. But that's my preference.

**Pull request merged**